hwproj.front/src/components/Homeworks/AddHomework.tsx

```diff
@@ -120,6 +122,7 @@ const AddHomework: React.FC<IAddHomeworkProps> = (props) => {
                     }
                 />
                 <Tags tags={[]} courseId={props.id} onTagsChange={handleTagsChange}/>
+                {addHomeworkState.tags?.includes('Контрольная работа') && <Alert severity="info">Вы можете сгруппировать контрольные работы и переписывания с помощью дополнительного тега. Например, 'КР 1'</Alert>}
```

Comment:
```suggestion
                {addHomeworkState.tags?.includes(TestTag) && <Alert severity="info">Вы можете сгруппировать контрольные работы и переписывания с помощью дополнительного тега. Например, 'КР 1'</Alert>}
```

https://github.com/InteIIigeNET/HwProj-2.0.1/blob/8608b0e8b62b9534cd4d52669cb3208e97d1476c/hwproj.front/src/components/Common/HomeworkTags.tsx#L4